public/src/components/admin_button_objs.jsx

```diff
@@ -0,0 +1,67 @@
+export default {
+  view_edit(props) {
+    const { thisPage } = props;
+    let target;
+    if (thisPage === 'Question') {
```

Comment:
Question: is there a reason we did not want to lift this logic up to the parent component?